src/string.cr

```diff
@@ -1585,7 +1585,7 @@ class String
   # ```
   # "aabbcc".tr("abc", "xyz") # => "xxyyzz"
   # "aabbcc".tr("abc", "x")   # => "xxxxxx"
-  # "aabbcc".tr("a", "xyz")   # => "xxbbcc"
+  # "aabbcc".tr("a", "xyz")   # => "xyzxyzbbcc"
```

Comment:
Don't want to derail this... but isn't that actually a bug in the `tr` implementation? Or did we intentionally diverge from Ruby's behaviour here?

Reply:
I don't know what specification is right. I'll create an issue about this. Thanks! 😄

Reply:
Oh, MakeNowJust fixed quickly! I will revert this. 😸